NEWS

```diff
@@ -76,13 +76,38 @@ NEWS - user visible changes				-*- outline -*-
    the following newly supported reference-formats (in addition to
    FIXED/FREE):
 
-   COBOL85      Fixed-form format with enforcement of Area A (pending)
-   COBOLX       GCOS 7 extended format
-   CRT          ICOBOL Free-form format
-   TERMINAL     ACUCOBOL-GT Terminal format
-   VARIABLE     Micro Focus Variable Fixed-form format
-   XCARD        ICOBOL xCard (extended card) format
-   XOPEN        X/Open Free-form format
+   COBOL85              Fixed-form format with enforcement of Area A
+   COBOLX		GCOS 7 extended format
+   CRT			ICOBOL Free-form format
+   TERMINAL		ACUCOBOL-GT Terminal format
+   VARIABLE		Micro Focus Variable Fixed-form format
+   XCARD		ICOBOL xCard (extended card) format
+   XOPEN		X/Open Free-form format
+
+   These formats come with Area A enforcement, that checks whether
+   division, section, and paragraph names start in Area A (i.e, before
+   margin B), and so do level indicators FD, SD, and RD, and level
+   numbers 01, 77, and 78.  The underlying checks are enabled or
+   disabled by default using dialect option `areacheck`, and then with
+   `>>AREACHECK` and `>>NOAREACHECK` compiler directives.
+
+   Area A enforcement additionally enables detection of missing
+   periods before paragraph and section names, configurable with
+   option `missing-period`.
+
+   As a result of reference format being a dialect option, Area A
+   enforcement is available for the following dialects:
+
+   BS2000 COBOL
```

Comment:
BS2000 explicit lists the areacheck (without option to disable it) in its docs; I've found nothing about Realia (I don't have a Language Reference and the User Reference have no entry for reference-format or rules for defining sections); RM-COBOL Language Reference also list this explicit check, so at least RM and BS2000 should have that enabled by default.

I'd actually go the easy way and say it is enabled by default (of course also adjust in all three of these missing) and only note the dialects, comma-separated (maybe even as their "std" abbreviations, but this may look bad, not tested).
If someone reports that this was an error for Realia we can adjust this later.